docs/13-walkthroughs/6-publish-and-deploy.md

```diff
@@ -0,0 +1,172 @@
+# located here: https://github.com/defenseunicorns/zarf/blob/main/examples/helm-oci-chart/zarf.yaml
+$ cat <<EOF > zarf.yaml
+kind: ZarfPackageConfig
+metadata:
+  name: helm-oci-chart
+  description: Deploy podinfo using a Helm OCI chart
+  # Note: In order to publish, the package must have a version
+  version: 0.0.1
+
+components:
+  - name: helm-oci-chart
+    required: true
+    charts:
+      - name: podinfo
+        version: 6.3.3
+        namespace: helm-oci-demo
+        url: oci://ghcr.io/stefanprodan/charts/podinfo
+    images:
+      - "ghcr.io/stefanprodan/podinfo:6.3.3"
+EOF
+```
+
+Create the package locally:
+
+```bash
+# Create the package (interactively)
+$ zarf package create .
+# Make these choices at the prompts:
+# Create this Zarf package? Yes
+# Please provide a value for "Maximum Package Size" 0
+```
+
+Then publish the package to the registry:
+
+```bash
+# Your package tarball may be named differently based on your machine's architecture
+$ zarf package publish zarf-package-helm-oci-chart-arm64-0.0.1.tar.zst oci://$REPOSITORY_URL
+
+...
+
+  •  Publishing package to $REPOSITORY_URL/helm-oci-chart:0.0.1-arm64
+  ✔  Prepared 14 layers
+  ✔  515aceaacb8d images/index.json
+  ✔  4615b4f0c1ed zarf.yaml
+  ✔  1300d6545c84 sboms.tar
+  ✔  b66dbb27a733 images/oci-layout
+  ✔  46564f0eff85 images/blobs/sha256/46564f0...06008f762391a7bb7d58f339ee
+  ✔  4f4fb700ef54 images/blobs/sha256/4f4fb70...b5577484a6d75e68dc38e8acc1
+  ✔  6ff8f4799d50 images/blobs/sha256/6ff8f47...4bc00ec8b988d28cef78ea9a5b
+  ✔  74eae207aa23 images/blobs/sha256/74eae20...fcb007d3da7b842313f80d2c33
+  ✔  a9eaa45ef418 images/blobs/sha256/a9eaa45...6789c52a87ba5a9e6483f2b74f
+  ✔  8c5b695f4724 images/blobs/sha256/8c5b695...014f94c8d4ea62772c477c1e03
+  ✔  ab67ffd6e92e images/blobs/sha256/ab67ffd...f8c9d93c0e719f6350e99d3aea
+  ✔  b95c82728c36 images/blobs/sha256/b95c827...042a9c5d84426c1674044916d4
+  ✔  e2b45cdcd8bf images/blobs/sha256/e2b45cd...000f1bc1695014e38821dc675c
+  ✔  79be488a834e components/helm-oci-chart.tar
+  ✔  d8399f7b56ca [application/vnd.unknown.config.v1+json]
+  ✔  aed84ba183e7 [application/vnd.oci.image.manifest.v1+json]
+  ✔  Published $REPOSITORY_URL/helm-oci-chart:0.0.1-arm64 [application/vnd.oci.image.manifest.v1+json]
+
+  •  To inspect/deploy/pull:
+  •  zarf package inspect oci://$REPOSITORY_URL/helm-oci-chart:0.0.1-arm64 --insecure
+  •  zarf package deploy oci://$REPOSITORY_URL/helm-oci-chart:0.0.1-arm64 --insecure
+  •  zarf package pull oci://$REPOSITORY_URL/helm-oci-chart:0.0.1-arm64 --insecure
+```
+
+:::note
+
+The name and reference of this OCI artifact is derived from the package metadata, e.g.: `helm-oci-chart:0.0.1-arm64`
+
+To modify, edit `zarf.yaml` and re-run `zarf package create .`
+
+:::
+
+## Inspect Package
+
+Inspecting a Zarf package stored in an OCI registry is the same as inspecting a local package and has the same flags:
+
+```yaml
+$ zarf package inspect oci://$REPOSITORY_URLhelm-oci-chart:0.0.1-arm64
```

Comment:
This is not expected I am assuming?
![image](https://user-images.githubusercontent.com/3977569/223591927-a38722f7-b1c8-4707-9b2d-f7509297d6bc.png)